### PR TITLE
Python math module math.atan

### LINKED ISSUE #5080: [Topic Entry] Python math-module math.atan()
### Reason/inspiration (optional)

We would like a new term entry on the `math.atan()` method of the [`math-module`](https://www.codecademy.com/resources/docs/python/math-module) concept in Python. The entry should go in a new file under `docs/content/python/concepts/math-module/terms/math-atan/math-atan.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Topic Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by don-mo**
Codecademy username: jJPOXFRED

### Description

Created docs\content\python\concepts\math-module\terms\math-atan\math-atan.md and I am completing the git and github course. I love github!

### Issue Solved

Closes #5080

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-atan/math-atan.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Data Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`math.atan()`** returns the inverse tangent of a number.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/math-module/terms/math-atan/math-atan.md:15]:
```suggestion
The **`math.atan()`** returns the inverse tangent of a number.
```
You can make the introduction more brief. It must be at least 160 - 170 characters long

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @don-mo ,
The entry is nicely written, I have suggested a change to you please make it asap.

Thanks,
Savi

File: content/python/concepts/math-module/terms/math-atan/math-atan.md (hunk 1 above)
Comment (don-mo) [reply content/python/concepts/math-module/terms/math-atan/math-atan.md:15]: You didn't make any changes to my file though. Do you mean I should make it longer and extend it to at least 160-170 characters long? @SaviDahegaonkar

**Comment by SaviDahegaonkar** [comment]
Yeah, exactly I mean the same. Please make it long at least 160-170
characters.

Thanks,
Savi

**Comment by don-mo** [comment]
@SaviDahegaonkar done I think. extended past 170 characters. let me know if there needs to be more changes

File: content/python/concepts/math-module/terms/math-atan/math-atan.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Data Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`math.atan()`** returns the arctangent of a number. If the tangent of a number x gives you n, then the arctangent of n gives you the number x. The number n can be is all real numbers. You can think of n as the slope of the line (opposite over adjacent or y/x) and x to be the degrees.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/math-module/terms/math-atan/math-atan.md:15]:
```suggestion
The **`math.atan()`** returns the arctangent of a number. If the tangent of a number x gives you n, then the arctangent of n gives you the number x. The number n can be all real numbers. You can think of n as the slope of the line (opposite over adjacent or y/x) and x to be the degrees. 
```
The result of `math.atan( )` is in radians and not in degrees, so please make this edit.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @don-mo , I have suggested a change to you. Please make it.

Thanks,
Savi

**Comment by don-mo** [comment]
@SaviDahegaonkar made changes!!

File: content/python/concepts/math-module/terms/math-atan/math-atan.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+The **`math.atan()`** returns the arctangent of a number. If the tangent of a number x gives you n, then the arctangent of n gives you the number x. The number n can be all real numbers. You can think of n as the slope of the line (opposite over adjacent or y/x) and x to be the radians.
+
+## Syntax
+
+```pseudo
+math.atan(n)
+```
+
+The `math.atan()` function returns the inverse tangent where the result will coincide whithin -π/2 and π/2 radians (the angle), representing the inverse tangent or arctangent of the value `n`.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/math-module/terms/math-atan/math-atan.md:23]:
```suggestion
The `math.atan()` function returns the inverse tangent where the result will coincide within -π/2 and π/2 radians (the angle), representing the inverse tangent or arctangent of the value `n`.
```

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @don-mo ,
I have a single grammatical change for you. Please make it asap.

Thanks,
Savi

**Review by SaviDahegaonkar — APPROVED** [review]

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @don-mo 😄 

The entry is ready to be merged! 🚀

**Comment by don-mo** [comment]
@mamtawardhani @SaviDahegaonkar I see I'm anonymous as a creditor. Would there be any way I can put my name on the document?

Thanks,
Don

**Comment by mamtawardhani** [comment]
Hey @don-mo, you can follow the following steps:

1. Go to your Codecademy profile dashboard
2. Click on your profile image in the top-right corner and choose 'Profile'
3. Then, click 'Edit Profile'
4. In the 'GitHub Username' field, add your username (without the @)
5. Click 'Save Changes'

**Comment by don-mo** [comment]
But I did that already though. My username is jJPOXFRED

[image: image.png]

**Pull request merged**